### ISSUE TITLE
"Login authentication failed" error

### ISSUE THREAD

**Issue by simbz33** [NorthernMan54/homebridge-tcc#72]
Hi there, 

I tried adding the homebridge accessory to my home app, and got this error on my iPhone:
"Accessory already added. To add this accessory to your home, you'll need to reset it first."

I have tried restarting the homebridge server and ui, and restarting my iPhone. The server is setup on a Mac mini. The two TCC devices I have are RTH9580WF and the RTH6580WF1001. Both are online and working in the portal. I used the same login info in the plugin, just copy and pasted it over. Here's the relevant log output I think:

```
Wed, 17 Jun 2020 22:58:03 GMT tcc-lib Queue: Working on item #26.  Size: 1  Pending: 0
Wed, 17 Jun 2020 22:58:04 GMT tcc-lib ERROR: Login Failed AuthenticationFailed <?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns="http://services.alarmnet.com/Services/MobileV2/"><soap:Body><AuthenticateUserLogin><username></password><applicationID>357568d9-38ff-4fda-bfe2-46b0fa1dd864</applicationID><applicationVersion>2</applicationVersion><uiLanguage>Default</uiLanguage></AuthenticateUserLogin></soap:Body></soap:Envelope>
Wed, 17 Jun 2020 22:58:04 GMT tcc-lib pollThermostat Error: Error: ERROR: Login FailedAuthenticationFailed
    at /usr/local/lib/node_modules/homebridge-tcc/lib/tcc.js:181:20
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[6/17/2020, 3:58:04 PM] [Thermostat] ERROR: pollDevices Error: ERROR: Login FailedAuthenticationFailed
Wed, 17 Jun 2020 22:59:03 GMT tcc-lib Queue: Working on item #27.  Size: 1  Pending: 0
Wed, 17 Jun 2020 22:59:04 GMT tcc-lib ERROR: Login Failed AuthenticationFailed <?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns="http://services.alarmnet.com/Services/MobileV2/"><soap:Body><AuthenticateUserLogin><username></password><applicationID>357568d9-38ff-4fda-bfe2-46b0fa1dd864</applicationID><applicationVersion>2</applicationVersion><uiLanguage>Default</uiLanguage></AuthenticateUserLogin></soap:Body></soap:Envelope>
Wed, 17 Jun 2020 22:59:04 GMT tcc-lib pollThermostat Error: Error: ERROR: Login FailedAuthenticationFailed
    at /usr/local/lib/node_modules/homebridge-tcc/lib/tcc.js:181:20
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[6/17/2020, 3:59:04 PM] [Thermostat] ERROR: pollDevices Error: ERROR: Login FailedAuthenticationFailed
```

**Comment by NorthernMan54** [comment]
Thanks for sharing a very well laid out issue, I was able to identify the issue and if you update to the latest it should be fine.

**Comment by simbz33** [comment]
Thanks for the update, that did fix the login issue I was seeing in the logs, but it still gives me the same error of "already added" in the home app on iOS.

Even when I deleted the home and created a new one on my phone, it successfully added the homebridge, but I see none of the device controls or devices show up, and it still gives me the same error as above when trying to add again. I do see my HomePod that's ready to be added.

I tried adding by code manually and by scanning the barcode in homebridge ui, still same thing :-(
